azure/postgres_firewall.sh

```diff
@@ -0,0 +1,72 @@
+# Postgres Servers firewall related functions
+
+# azure_postgres_firewall_rule_create creates a new rule
+# for the firewall of a server of "managed Postgres".
+# `name` is the name of the rule.
+# `group` is the name of resource group.
+# `servername` is the name of the Postgres server.
+# `startip` is the start IP address of the firewall rule. (IPV4 format)
+# `endip` is the end IP address of the firewall rule. (IPV4 format)
```

Comment:
You're right. I will remove the return of $out in the end. You think it should have something in the "docs" about returning an error?